src/AutoSplitImage.py

```diff
@@ -98,6 +121,39 @@ def __init__(self, path: str):
         else:
             self.image_type = ImageType.SPLIT
 
+    def __parse_text_file(self, path: str):
+        if not TESSERACT_PATH:
+            error_messages.tesseract_missing(path)
+            return
+
+        with open(path, encoding="utf-8") as f:
+            data = toml.load(f)
+
+        self.texts = [text.lower().strip() for text in data["texts"]]
+        self.__rect = [
+            data["top_left"][0],
+            data["bottom_right"][0],
+            data["top_left"][1],
+            data["bottom_right"][1],
```

Comment:
Is there a reason to not use `left, right, top, bottom` as recommended here?https://github.com/Toufool/AutoSplit/pull/272/files/f1ba4108a91aa23ace77f002b279b9f985979924..d1dfff05bd16f4056e462c8a3aaa90cd388f258e#r1485635071

This is the only place in the code where we deal with a pair of coordinates (corners) so that's odd.